lib/agoneum/web/controllers/page_controller.ex

```diff
@@ -1,7 +1,16 @@
 defmodule Agoneum.Web.PageController do
   use Agoneum.Web, :controller
 
+  plug Guardian.Plug.EnsureAuthenticated, handler: __MODULE__
+
   def index(conn, _params) do
-    render conn, "index.html"
+    user = Guardian.Plug.current_resource(conn)
+    render conn, "index.html", user: user
+  end
+
+  def unauthenticated(conn, _params) do
```

Comment:
I'd like to find a common location to place this so that I don't need to duplicate it across controllers